app/controllers/comments_controller.rb

```diff
@@ -47,7 +47,29 @@ def destroy
     end
   end
 
+  def like
+    toogle_reaction(true)
+  end
+
+  def dislike
+    toogle_reaction(false)
+  end
+
   private
+
+    def toogle_reaction(reaction_type)
+      @comment = Comment.find(params[:id])
+      existing_reaction = Reaction.find_by(user_id: current_user.id, likeable_id: @comment.id)
```

Comment:
its gives you one record use where if needed more than one record

Reply:
I need only one.

Reply:
Done.